### PR TITLE
Addresses #1759 NTR valve endothelial cell

### PR DESCRIPTION
**Pull request by bvarner-ebi**
Addresses #1759 NTR valve endothelial cell
#gogoeditdiff

Appears to contain some extra changes from the recently merged PR #1727. (EDIT: [these have been removed](https://github.com/obophenotype/cell-ontology/pull/1763#issuecomment-1344382357))

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -24511,7 +24524,7 @@ EquivalentClasses(obo:CL_1000143 ObjectIntersectionOf(obo:CL_0000160 ObjectSomeV
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25311230"^^xsd:string) obo:IAO_0000115 obo:CL_1000147 "A cell that is part of a cardiac valve."^^xsd:string)
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30796046"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_1000147 "heart valve cell"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_1000147 "cardiac valve cell"^^xsd:string)
-EquivalentClasses(obo:CL_1000147 ObjectIntersectionOf(obo:CL_0000000 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000946)))
+EquivalentClasses(obo:CL_1000147 ObjectIntersectionOf(obo:CL_0000003 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000946)))
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:24527]: See [this comment](https://github.com/obophenotype/cell-ontology/pull/1758#issuecomment-1339672528).

Comment (shawntanzk) [reply src/ontology/cl-edit.owl:24527]: hmmm this is odd given you didn't change this. If it's not your change, would you mind reverting this please? 
Thanks

Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:24527]: To clarify: I did make this change as noted in the linked comment.
The declared PATO, UBERON and PR are the parts I did not add.

Comment (shawntanzk) [reply src/ontology/cl-edit.owl:24527]: wait so do you want it to be native cell or cell?

Comment (shawntanzk) [reply src/ontology/cl-edit.owl:24527]: > The declared PATO, UBERON and PR are the parts I did not add.

This is strange, I always thought protege doesn't declare imports, not quite sure why this is all coming up. I vaguely rmbr there being some minor concerns with declaring imported terms (I think it has something to do with if they are declared but not used), but its not a HUGE issue. Let me check with other PRs and all and get back to you on this

Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:24527]: 'native cell'

Comment (shawntanzk) [reply src/ontology/cl-edit.owl:24527]: yeah I'm not sure why protege is declaring all these terms, almost certain we don't want them. Are you using the new beta release? 
Either way could you try removing them please? (would do it but running a import on another branch atm)

**Comment by dosumis** [comment]
Declarations can happen depending on how you add new terms for import.  If you paste the IRIs straight into Protege, that => a declaration in the edit file.

**Comment by bvarner-ebi** [comment]
> Declarations can happen depending on how you add new terms for import. If you paste the IRIs straight into Protege, that => a declaration in the edit file.

Yes, I have done that for other tickets as it is listed as the first option [here](https://oboacademy.github.io/obook/howto/update-import/#importing-a-new-term). The page says this method is "to be avoided" but does not give a reason and it has consistently worked for prior tickets.

The declarations appearing in this PR, however, are a surprise since I did not add them. If I had to guess, they may be coming from PR https://github.com/obophenotype/cell-ontology/pull/1727 since they all seem to appear there.
Regardless, I'll delete the declarations in this PR and then merge.

**Comment by dosumis** [comment]
{ Off topic for this PR but: adding IRIs directly has the great advantage that it can be used instantly without having to wait for someone to help or disrupting your editing flow run a slow import update.  I wonder if we should tolerate it and have a clean-up job that removes stray declarations from the edtor's file. Alternatively maybe we could have some super minimal temp import update (say  IRI + label only) that editors can use rapidly while working on a PR, postponing a full update to later  - Maybe add to agenda for Monday tech call? }

**Comment by bvarner-ebi** [comment]
The following new term has been added to CL and will be available in the next release:

CL:4030033 'valve endothelial cell'

**Comment by shawntanzk** [comment]
Should this have the GO term attached to it like in https://github.com/obophenotype/cell-ontology/pull/1761?

**Comment by bvarner-ebi** [comment]
> Should this have the GO term attached to it like in #1761?

Not totally sure about this... the reference notes they secrete paracrine molecules, but I have not found a reference that explicitly states VECs **secrete extracellular matrix constituents**. Activated VICs are noted explicitly to secrete fibrous matrix proteins (collagen), which is why the GO axiom was added to that term.

From https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4485434/:

"VECs also secrete paracrine factors, including nitric oxide, endothelin-1, and C-type natriuretic peptide. Most biochemical factors secreted by VECs (except endothelin-1) function to reduce the pathogenic phenotypes of VICs."

vs.

"In addition, VICs secrete fibrous matrix proteins, matrix metalloproteinases (such as MMP1 and MMP2), biochemical factors, and can differentiate into myofibroblasts or osteoblast-like cells or undergo cell death-mediated calcification"

**Pull request merged**